src/objects/business/VersionedObject.ts

```diff
@@ -0,0 +1,27 @@
+import { PropertiesOf } from 'crunchDB/objects/utilities';
```

Comment:
VersionedObject is not something that CrunchDB should worry about. It should be provided with the current schema on initialization.